crates/consensus/src/validation.rs

```diff
@@ -259,15 +241,16 @@ pub fn validate_header_regarding_parent(
     }
 
     // difficulty check is done by consensus.
-    if chain_spec.paris_status().block_number() > Some(child.number) {
-        // TODO how this needs to be checked? As ice age did increment it by some formula
-    }
+    // TODO(onbjerg): Unsure what the check here is supposed to be, but it should be moved to
+    // [BeaconConsensus]. if chain_spec.paris_status().block_number() > Some(child.number) {
+    //    // TODO how this needs to be checked? As ice age did increment it by some formula
+    //}
```

Comment:
is this relevant? what check needs to exist here, given that we now check total iff inside of the HeadersTD stage? cc @rkrasiuk